### PR TITLE
Update pan06_1a.xml

### PR DESCRIPTION
**Pull request by rom169**
Corrected Philio PAN06-1A:
Parameter 1 default and order.
Parameter 4 type.

Added:
Philio PAN11-1B
Everspring AN179 
Everspring AD146

Regards
rom169

### CODE REVIEW

**Comment by rom169** [comment]
This changes will solve some bugs related to configuration parameters of Philio PAN06-1A.
regards
rom169

**Comment by rom169** [comment]
Add Philio PAN11-1B to database.

**Comment by rom169** [comment]
Added Everspring AN179 to database.

**Comment by rom169** [comment]
Added Everspring AD146 to database.

File: bundles/binding/org.openhab.binding.zwave/database/everspring/ad146.xml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+		<Class><id>0x86</id></Class>
+	</CommandClasses>
+	<Configuration>
+		<Parameter>
+			<Index>1</Index>
+			<Type>short</Type>
+			<Minimum>0</Minimum>
+			<Maximum>255</Maximum>
+			<Default>255</Default>
+			<Size>2</Size>
```
Comment (cdjackson) [inline bundles/binding/org.openhab.binding.zwave/database/everspring/ad146.xml]: Are you sure the size is correct? It has a max value of 255 which just seems inconsistent with a 2 byte value (it still might be correct, but looks wrong, so just checking)

File: bundles/binding/org.openhab.binding.zwave/database/everspring/an179.xml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+		<Class><id>0x86</id></Class>
+	</CommandClasses>
+	<Configuration>
+		<Parameter>
+			<Index>1</Index>
+			<Type>short</Type>
+			<Minimum>0</Minimum>
+			<Maximum>255</Maximum>
+			<Default>255</Default>
+			<Size>2</Size>
```
Comment (cdjackson) [inline bundles/binding/org.openhab.binding.zwave/database/everspring/an179.xml]: Same comment about size - this has a max value of 255 so just checking the size is really 2 bytes?

Comment (rom169) [inline bundles/binding/org.openhab.binding.zwave/database/everspring/an179.xml]: Yes it is 2 byte according to the documentation (page 3 top left):
http://products.z-wavealliance.org/MarketCertification/File?folder=&filename=MarketCertificationFiles/1058/Manual_AN179_20140808.pdf

File: bundles/binding/org.openhab.binding.zwave/database/everspring/ad146.xml (hunk 1 above)
Comment (rom169) [inline bundles/binding/org.openhab.binding.zwave/database/everspring/ad146.xml]: Yes it is 2 byte according to the documentation (page 2 bottom left):
http://products.z-wavealliance.org/MarketCertification/File?folder=&filename=MarketCertificationFiles/1057/Manual_AD146-0_20140728.pdf
Reviewing this today I found, that I have forgotten the 5th parameter of this device. Will create a pull request in a minute for this.

**Comment by cdjackson** [comment]
Thanks.

**Pull request merged**